Recipes/Recipes.API/Program.cs

```diff
@@ -13,36 +19,47 @@
     app.UseSwaggerUI();
 }
 
-app.MapGet("/recipes", () =>
+app.MapGet("/recipes", async (RecipesDatabaseContext dbContext) =>
+{
+    var recipe = new RecipeEntity()
```

Comment:
Won't we save many copies of similar RecipeEntity items like this, or was it the plan?

Reply:
Feel free to solve on your own.

Reply:
Yeah that's what I figured as well but assumed this is just for testing, since the correct endpoints will be implemented in #9

Reply:
Yes, this is just to showcase that the DB works